jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java

```diff
@@ -245,31 +247,31 @@ public JibContainerBuilder addEnvironmentVariable(String name, String value) {
    * <p>This is similar to <a href="https://docs.docker.com/engine/reference/builder/#volume">{@code
    * VOLUME} in Dockerfiles</a>.
    *
-   * @param volumes the list of directory paths on the container filesystem to set as volumes
+   * @param volumes the directory paths on the container filesystem to set as volumes
    * @return this
    */
-  public JibContainerBuilder setVolumes(List<AbsoluteUnixPath> volumes) {
-    containerConfigurationBuilder.setVolumes(volumes);
+  public JibContainerBuilder setVolumes(Set<AbsoluteUnixPath> volumes) {
+    containerConfigurationBuilder.setVolumes(new HashSet<>(volumes));
```

Comment:
`new HashSet` isn't necessary since `ContainerConfiguration.Builder#setVolumes` already does the `new HashSet`.